SPECIFICATION.md

```diff
@@ -0,0 +1,293 @@
+# Playbook specification
+
+This document describes the exact format required by the Ansible playbook verifier.
+
+```yaml
+---
+- name: Example playbook
+  hosts: localhost
+  become: yes
+  vars:
+    insights_signature_exclude: /hosts,/vars/insights_signature
+    insights_signature: !!binary |
+      base64 binary blob containing the embedded GPG signature
+  tasks:
+    - name: Display debug message
+    - ansible.builtin.debug:
+        msg: The signed playbook says hello!
+```
+
+## Only one playbook may be present
+
+In case there are multiple playbooks present in the file, only the first one will be verified. Any other playbook is explicitly ignored.
+
+**Warning:** Even though only the first playbook is verified, all of them may be emitted to standard output after the verification.
+
+<details>
+
+<summary>Example</summary>
+
+```yaml
+# valid
+- name: Example playbook
+  hosts: localhost
+  ...
+```
+```yaml
+# invalid
```

Comment:
hmm "invalid" IMHO is not correct: this yaml file will be processed, and only the first play will be verified

my suggestion here is to drop this "invalid" comment, and add something like
```yaml
# this and any playbook that follows is not verified
```
right before the 2nd play